### ISSUE TITLE
Missing _get_interval_dir function in packaging/installer/functions.sh cause auto-updating not working

### ISSUE THREAD

**Issue by marcingarski** [netdata/netdata#9990]
##### Bug report summary

After #9598 Netdata stopped auto-updating. The cause of this is missing `_get_interval_dir` function which is called on line 967 (also on 969 for message display) that creates symlink to netdata-updater. The function that exist is `_get_intervaldir`.

##### OS / Environment

```
Linux kvm6 5.4.44-2-pve #1 SMP PVE 5.4.44-2 (Wed, 01 Jul 2020 16:37:57 +0200) x86_64 GNU/Linux
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 10 (buster)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="10"
/etc/os-release:VERSION="10 (buster)"
/etc/os-release:VERSION_CODENAME=buster
/etc/os-release:ID=debian
/etc/os-release:HOME_URL="https://www.debian.org/"
/etc/os-release:SUPPORT_URL="https://www.debian.org/support"
/etc/os-release:BUG_REPORT_URL="https://bugs.debian.org/" 
```

##### Netdata version

 1.25.0-41-nightly

##### Component Name

packaging
##### Steps To Reproduce

1. Wait for auto-update or run
2. bash <(curl -Ss https://my-netdata.io/kickstart.sh)
```
 --- Check if we must enable/disable the netdata updater tool ---
./packaging/installer/functions.sh: line 967: _get_interval_dir: command not found
./packaging/installer/functions.sh: line 969: _get_interval_dir: command not found
Auto-updating has been enabled through cron, updater script linked to /netdata-updater 
```
##### Expected behavior

Auto-updating is working

**Comment by Ferroin** [comment]
PR opened to fix this (#9992). Once it's merged, you'll need to wait for the next nightly to go out (I plan to merge it today, which means it should be in tonight's nightly and therefore available after 2020-09-26 02:00 UTC).

Once it's in the nightlies, the next auto-update that would occur should work correctly, or you can manually trigger an update by running the kickstart script.

**Comment by marcingarski** [comment]
> PR opened to fix this (#9992). Once it's merged, you'll need to wait for the next nightly to go out (I plan to merge it today, which means it should be in tonight's nightly and therefore available after 2020-09-26 02:00 UTC).

Thanks, for resolving the issue. 

> Once it's in the nightlies, the next auto-update that would occur should work correctly, or you can manually trigger an update by running the kickstart script.

I had to manually trigger an update because the auto-update stopped working after #9598 (the symlink was not created).
